index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Document</title>
+    <script type="text/javascript" src="dist/envspec.js"></script>
+     <link rel="stylesheet" href="dist/styles.css">
+  </head>
+
+  <body>
+    <script type="text/javascript" src="dist/demoPage.js"></script>
+    <p><h1>.ENV.SPEC demo page</h1></p>
```

Comment:
1. Remove the parent `<p>` tag, as `<h1>` tags cannot be put inside paragraphs.
1. Rename `.ENV.SPEC` to `.env.spec`